webapp/src/js/bootstrapper/server-side-purge.js

```diff
@@ -0,0 +1,170 @@
+const utils = require('./utils');
```

Comment:
Maybe this file should just be called `purger.js` now that we only have one?

Reply:
Yea, will do. I had 2 files at some point (the old and the new).